public/app.json

```diff
@@ -23,7 +23,7 @@
         }
     },
     "runtime": {
-        "arguments": "--noerrdialogs --frameStrategy=frames",
+        "arguments": "--noerrdialogs --frameStrategy=frames --enable-aggressive-domstorage-flushing",
```

Comment:
This extra arg was done during preliminary testing, is it still needed after other modification?

Reply:
I like this anyway, we do other things that require local storage to be stable